docker/httpd/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM alpine
+RUN mkdir -p /var/www/html/
+COPY . /var/www/html/
+WORKDIR /var/www/html/
+
+RUN apk add --no-cache yarn && \
+    yarn install && \
+    yarn run encore production
+
+FROM httpd:2.4-alpine
+MAINTAINER Bart Geesink (bart.geesink@surfnet.nl)
+RUN mkdir -p /var/www/html/
+COPY --from=0 /var/www/html/web/. /var/www/html/web/
```

Comment:
Personally I would use `--from=node-build` and then use `FROM node as node-build`  in the referenced build stage.

Reply:
I've added your suggestions